paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -148,7 +151,7 @@ inference_analysis_api_test_with_fake_data(test_analyzer_mobilenet_depthwise_con
 if(WITH_MKLDNN)
   set(INT8_DATA_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8")
```

Comment:
please change the INT8_DATA_DIR if you change the data. Otherwise, since INT8_DATA_DIR is already existed ON CI, it will be error.

Reply:
With this patch, we keep the `data.bin` file name but change the archive name into `imagenet_val_100_tail.tar.gz`, so there should be no conflict of archive names, or might there be a conflict between several builds?

Reply:
if INT8_DATA_DIR is already exised ON CI, it will not download `imagenet_val_100_tail.tar.gz`

Reply:
do you mean that several CI builds share the same directory with the dataset?

Reply:
Our CI cache the `INFERENCE_DEMO_INSTALL_DIR` for speedup.

Reply:
set(INT8_DATA_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8v2") or other name

Reply:
Done.